{
  try {
    serverInstance.shutDown();
  }
 catch (  final Exception e) {
    LOGGER.error("Caught exception",e);
  }
}
